youtube_dl/extractor/jamendo.py

```diff
@@ -35,6 +35,19 @@ class JamendoIE(JamendoBaseIE):
                         /track/(?P<id>[0-9]+)/(?P<display_id>[^/?#&]+)
                     '''
     _TESTS = [{
+        'url': 'https://www.jamendo.com/track/1311140/221',
+        'md5': '9a73cd099a5453244bf73044442c8330',
+        'info_dict': {
+            'id': '1311140',
+            'display_id': '221',
+            'ext': 'flac',
+            'title': 'Duration of life - 221',
+            'artist': 'Duration of life',
+            'track': '221',
+            'duration': 141,
+            'thumbnail': r're:^https?://.*\.jpg'
+        }
+    }, {
```

Comment:
Remove duplicate tests.

Reply:
I'll push it when I've fixed the duration extraction.